tensorflow/docs_src/tutorials/seq2seq.md

```diff
@@ -1,5 +0,0 @@
-# Sequence-to-Sequence Models
```

Comment:
Are we booting the nmt tutorial? Don't we prefer to include stubs for files that are pulled during site-build?

Reply:
Readded